### DISCUSSION TITLE
Add support for Sidekick browser

### DISCUSSION THREAD

**Discussion by Liquefy7822** [will-stone/browserosaurus#627] [Ideas]
Sidekick browser is not supported and does not appear

**Comment by will-stone** [comment]
It should do: https://github.com/will-stone/browserosaurus/blob/39b766f56c3e135ee7b6497be90ca5c8816bdd14/src/config/apps.ts#L111

Do you have it installed under `/Applications/Sidekick.app`?

**Comment by Liquefy7822** [comment]
Yup it does. Thank!

On Wednesday, March 8, 2023, Will Stone ***@***.***> wrote:

> It should do: https://github.com/will-stone/browserosaurus/blob/
> 39b766f56c3e135ee7b6497be90ca5c8816bdd14/src/config/apps.ts#L111
>
> Do you have it installed under /Applications/Sidekick.app?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/will-stone/browserosaurus/discussions/627#discussioncomment-5241900>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A6LJ6T4WMYF63O5XD5WK4PDW3CEQBANCNFSM6AAAAAAVTW4Y2Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> com>
>


-- 

Best regards,
[image: Emmanuel Yee Hong Wei]
Emmanuel Yee Hong Wei
Online privacy enthusiast & Apple Teacher

sites.google.com/s2022.ssts.edu.sg/emmanuelyeehongwei/home

***@***.***
[image: linkedin] <https://www.linkedin.com/in/emmanuelyee>

IMPORTANT: The contents of this email and any attachments are confidential.
They are intended for the named recipient(s) only. If you have received
this email by mistake, please notify the sender immediately and do not
disclose the contents to anyone or make copies thereof.
[image: Green] Please consider the environment before printing this e-mail!

Create your own email signature
<https://www.wisestamp.com/create-own-email-signature/?utm_source=promotion&utm_medium=signature&utm_campaign=create_your_own&srcid=>
